.github/workflows/release.yml

```diff
@@ -0,0 +1,25 @@
+name: Prepare release
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types:
+      - closed
```

Comment:
I think we don't need to run this workflow when a PR is closed. It might be just a test PR or something we didn't merge.